plugins/jobs/src/js/jobsMenu.js

```diff
@@ -1,5 +1,32 @@
+import StringUtil from "#SRC/js/utils/StringUtil";
+import UserActions from "#SRC/js/constants/UserActions";
 import jobsRunNow from "./jobsRunNow";
+import jobsToggleSchedule from "./jobsToggleSchedule";
 
-export default function(jobId) {
-  return [jobsRunNow(jobId)];
+export default function(job, customActionHandlers) {
```

Comment:
Can we name the function? In the past that has helped me with error messages, as it leaves a paper trail